requirements.txt

```diff
@@ -1,4 +1,5 @@
 bcrypt==3.1.3
+cryptography==1.7.1
```

Comment:
I think that the reason changing this throws an error is because the common library also imports cryptography and it also uses 1.7.1. We'll probably need to update both if we want to change it